packages/eslint-config/index.js

```diff
@@ -222,6 +222,10 @@ const unicornRules = {
   'unicorn/prefer-node-protocol': 'off',
   // Conflict with `@silverhand/fp/no-mutating-methods` when calling `.slice().sort()`
   'unicorn/prefer-spread': 'off',
+  'unicorn/no-await-expression-member': 'off',
+  'unicorn/template-indent': 'off',
+  'unicorn/prefer-export-from': 'off',
+  'unicorn/prefer-code-point': 'warn',
```

Comment:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-code-point.md
also could update our code?

![image](https://user-images.githubusercontent.com/36393111/146304079-99221566-2459-432a-bdde-67beca73ab92.png)

@gao-sun this is the only placing using `charCodeAt`

Reply:
I tried to replace .charCodeAt() with .codePointAt(), but the return type of charCodeAt() is number or NaN while the return type of charCodeAt() is number or undefined. Hence the lint-rule-checker throw error that operand '+' can not work between a number type variable (accumulator) and undefined variable even if I did type-check. Any idea that I can follow to walk around?

Reply:
cool, let me double check

Reply:
![image](https://user-images.githubusercontent.com/36393111/146318451-b8551e6b-beab-4090-90d5-a65ca337000f.png)
maybe this approach?